elastic/kibana
# Single list of rule types in observability without grouping

Closes #166136

## Summary

This PR removes the groups in the rules list in the observability.

|With disabled rule|Without disabled rule|
|---|---|
|![image](https://github.com/elastic/kibana/assets/12370520/ace3e46d-f125-4b66-848a-9b3fd06d5ea8)|![image](https://github.com/elastic/kibana/assets/12370520/aa89d441-6bbd-46d1-8753-bcddcd03c518)|

## 🧪 How to test
- Check the rule type list in observability > rules page
- Check the filter works as before
